src/framework/mpas_attlist.F

```diff
@@ -253,6 +261,213 @@ subroutine mpas_add_att_text(attList, attName, attValue, ierr)!{{{
 
    end subroutine mpas_add_att_text!}}}
 
+!***********************************************************************
+!
+! routine mpas_modify_att_text
+!
+! > \brief  MPAS modify text attribute routine
+! > \author Matthew Dimond
+! > \date   06/27/23
+! > \details
+! > This routine modifies a text attribute in the attribute list,
+! > and returns a 1 in ierr if the attribute is not found, or the attribute
+! > has a type incompatible with attValue.
+!
+!----------------------------------------------------------------------
+  subroutine mpas_modify_att_text(attList, attName, attValue, ierr)!{{{
+
+    implicit none
+
+    type (att_list_type), pointer :: attList !< Input/Output: Attribute List
+    character (len=*), intent(in) :: attName !< Input: Att. name to modify
+    character (len=*), intent(in) :: attValue !< Input: Updated Att. value
+    integer, intent(out), optional :: ierr !< Output: Error flag
+
+    type (att_list_type), pointer :: cursor
+
+    if (present(ierr)) ierr = 1
+
+      ! Traverese list looking for attName
```

Comment:
It's probably worth correcting the indentation here to match that of the code block below.

Also, here and in other places, "Traverese" -> "Traverse".